src/client/cli/cmd/transfer.cpp

```diff
@@ -163,9 +165,16 @@ std::vector<std::pair<std::string, fs::path>> cmd::Transfer::args_to_instance_an
     instance_entry_args.reserve(args.size());
     for (const auto& entry : args)
     {
+        // this is needed so that Windows absolute paths are not split at the colon following the drive letter
+        if (QRegularExpression{R"(^[A-Za-z]:[\\/].*)"}.match(entry).hasMatch())
+        {
+            instance_entry_args.emplace_back("", entry.toStdString());
+            continue;
+        }
+
         const auto source_components = entry.split(':');
         const auto instance_name = source_components.size() == 1 ? "" : source_components.first().toStdString();
-        const auto file_path = source_components.size() == 1 ? source_components.first() : source_components.at(1);
+        const auto file_path = source_components.size() == 1 ? source_components.first() : entry.section(':', 1);
```

Comment:
Is this meant to handle linux paths with colons in them? But then how do we know which colon is the splitting one. If we don't know we can handle such cases correctly, I wonder if we're not better off just rejecting them from the get go? Or are you reverting to what we did earlier?

Reply:
Yep, makes sense :+1: